ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/modals/cart/clientlibs/site/js/cart-actions.js

```diff
@@ -39,7 +38,6 @@ jQuery((function ($, ns, messages, cart, semanticModal, licenseModal) {
         e.stopPropagation();
 
         if (cart.contains(assetPath)) {
-            messages.show(MESSAGE_CART_EXISTS);
```

Comment:
This message cannot be set by page properties dialogs, so I removed it.  Im wondering if cart-exists got removed at some point when the "Add to Cart" changed to "Remove from Cart" for items in the cart.

Reply:
Yeh - originally we had no way to remove an item from the cart without going to the cart, so the "add to cart" button just stuck around even after it was added. This was a bit confusing so there was the "item already exists in cart" if you clickd it again ... but now that the button toggled between add/remove we dont really have the concept of adding something that was already added (since the button will now be remove)